includes/polling/os/ceraos.inc.php

```diff
@@ -1,5 +1,31 @@
 <?php
+/*
+ * LibreNMS
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
 
-$hardware = snmp_get($device, 'genEquipInventoryCardName.127', '-Osqnv', 'MWRM-UNIT-MIB');
-$serial = snmp_get($device, 'genEquipInventorySerialNumber.127', '-Osqnv', 'MWRM-UNIT-MIB');
-$version = snmp_get($device, 'genEquipMngSwIDUVersionsRunningVersion.1', '-Osqnv', 'MWRM-UNIT-MIB');
+$ceragon_type = snmp_get($device, 'sysObjectID.0', '-mSNMPv2-MIB -Oqv', '');
```

Comment:
If I don't use -mMIB-NAME, I get the following:
genEquipInventorySerialNumber.0: Unknown Object Identifier (Sub-id not found: (top) -> genEquipInventorySerialNumber)
I know it is unlikely that the numerical OID would change in eg IF-MIB but it is possible since the textual OID is canonical and I think it is far more possible that it will change in the ceragon MIBs in order to support future variants.